js/searchResultsView.js

```diff
@@ -4,60 +4,56 @@ define(function(require){
     var SearchAlgorithm = require('./search-algorithm');
 
     var replaceTagsRegEx = /\<{1}[^\>]+\>/g;
+    var replaceEscapedTagsRegEx = /&lt;[^&gt;]+&gt;/g;
     var replaceEndTagsRegEx = /\<{1}\/{1}[^\>]+\>/g;
+    var replaceEscapedEndTagsRegEx = /&lt;\/[^&gt;]+&gt;/g;
         
     var SearchResultsView = Backbone.View.extend({
 
         className : 'search-results inactive',
-        
-        initialize: function(options) {
 
-          this.listenTo(Adapt, 'drawer:empty', this.remove);    
-          this.listenTo(Adapt, 'search:termsFiltered', _.bind(this.updateResults, this));      
+        events: {
+          "click [data-id]":"navigateToResultPage"
+        },
+
+        initialize: function(options) {
+          this.listenTo(Adapt, {
+            'drawer:empty': this.remove,
+            'search:termsFiltered': _.bind(this.updateResults, this)
```

Comment:
I know it was already present - but I don't think it should be necessary to use `_.bind` here